### DISCUSSION TITLE
How to load Noto Sans Japanese fonts

### DISCUSSION THREAD

**Discussion by hayawata3626** [vanilla-extract-css/vanilla-extract#1191] [Q&A]
I am currently creating a UI library and am trying to load the following fonts using `globalFontFace`, but there are so many **.woff2** files to load that I don't think it is practical.(Reference: https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;500;700)
- Regular 400
- Medium 500
- Bold 700

In this case, is it better to install ttf files?
If there is another better way, please let me know.

**Comment by askoufis** [comment]
I don't think using ttf files will help you here, it's just a different (worse relative to woff2) file format.

Are you re-writing https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;500;700 using Vanilla Extract? If, so I would suggest abandoning that effort and just sticking the link tags from the [google fonts site](https://fonts.google.com/noto/specimen/Noto+Sans+JP) directly into your HTML. There's no practical benefit to handwriting all that CSS using Vanilla Extract.
